package.json

```diff
@@ -52,9 +53,9 @@
     "react-intl": "^5.7.0",
     "react-query": "^3.6.0",
     "react-router-dom": "^5.2.0",
+    "react-virtualized-auto-sizer": "*",
     "regenerator-runtime": "^0.13.3",
-    "sinon": "^7.1.1",
-    "react-virtualized-auto-sizer": "*"
+    "sinon": "^7.1.1"
```

Comment:
I understand this is just yarn reorganizing deps, but it is never save to depend on a `*` version. RVAS, in particular, has been the source of some really sneaky problems over the last few months ([STCOM-1148](https://issues.folio.org/browse/STCOM-1148)). If you notice this kind of thing in the future, please file a ticket to fix it. I filed [UIDEXP-325](https://issues.folio.org/browse/UIDEXP-325) for this one.